webpack.config.js

```diff
@@ -92,12 +98,58 @@ const createCssLoaderConfig = () =>
 
 const createCssExtractTextPlugin = () => new ExtractTextPlugin(CSS_FILENAME_OUTPUT_PATTERN);
 
+class PostCompilePlugin {
+  constructor(fn) {
+    this.fn = fn;
+  }
+
+  apply(compiler) {
+    compiler.plugin('done', (stats) => this.fn(stats));
+  }
+}
+
+const createStaticBuildPlugin = () => {
+  return new PostCompilePlugin(() => {
+    if (!BUILD_STATIC_DEMO_ASSETS || !fs.existsSync(OUT_DIR_ABS)) {
+      return;
+    }
+
+    const demosDirAbs = path.resolve('./demos');
+    const tmpDirAbs = fs.mkdtempSync(path.join(os.tmpdir(), 'mdc-web-demo-output-'));
+
+    const copyOptions = {
+      // eslint-disable-next-line no-unused-vars
+      filter: (src, dest) => {
+        return !/\.(scss|css.js)$/.test(src);
+      },
+    };
+
+    fsx.copySync(demosDirAbs, tmpDirAbs, copyOptions);
+    fsx.copySync(OUT_DIR_ABS, path.join(tmpDirAbs, DEMO_ASSET_DIR_REL), copyOptions);
+
+    if (!WRAP_CSS_IN_JS) {
+      glob.sync(path.join(tmpDirAbs, '**/*.html'))
+        .forEach((absPath) => {
+          const oldHtml = fs.readFileSync(absPath, {encoding: 'utf8'});
+          const newHtml = oldHtml.replace(/<script src="([^"]+\.css\.js[^"]*)"><\/script>/ig, (match, oldPath) => {
```

Comment:
Can't this whole `replace` call be reduced to this: 

```js
const newHtml = oldHtml.replace(
  /<script src="([^"]+\.css)\.js"><\/script>/ig,
  '<link rel="stylesheet" href="$1">');
```

* If you don't capture the `\.js` then you don't have to strip it manually
* I'm not sure why you need the `[^"]*` at the end at all; all instances of `.css.js` I see in our HTML files are immediately followed by `"`
* `$1` works within the replacement string to refer to the capture group

Reply:
_**Wow.**_ Derp. Updated. Thanks!

(I originally added the `[^"]*` at the end because I was planning to eventually replace Webpack's stupid CSS hot reloading with a query string param along the lines of
`?timestamp=${Date.now()}`. But now that I think about it, the search/replace doesn't need to worry about the query string, because it would be added dynamically by JS, not in the static HTML...)